cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -203,6 +204,11 @@ class EvaluatorCommand : CliktCommand(name = "evaluate", help = "Evaluate ORT re
         help = "Do not evaluate the script but only check its syntax. No output is written in this case."
     ).flag()
 
+    private val allowSourceCodeDownload by option(
+        "--allow-source-code-download",
```

Comment:
I don't think we need this if the API was better, by moving the source code download to a dedicated rule.

Reply:
I just had the idea of turning
```
fun ortResultRule(name: String, configure: OrtResultRule.() -> Unit) {
```
into 
```
fun ortResultRule(name: String, configure: OrtResultRule.() -> Unit, allowSourceCodeDownload: Boolean = false) {
```
Would this eliminate the need for a dedicated rule class? I'd still prefer not introducing a dedicated class but using OrtResultRule. However, if this is the only way to unblock this pull request then I will do it.  So if you insist on this dedicated class, maybe you have a good idea for a name? (I don't)

Reply:
Dropped this.